lib/controllers/project.js

```diff
@@ -21,779 +22,720 @@
 
-        switch(type) {
-          case Project.types.NODEJS:
-            cb(null, dir);
-            break;
-          case Project.types.METEOR:
-            includeModules = true;
-            modulus.io.print('Meteor project detected...');
+      switch (type) {
+        case Project.types.NODEJS:
+          return cb(null, dir);
+        case Project.types.METEOR:
+          includeModules = true;
+          print('Meteor project detected...');
 
-            projectController.demeteorize(dir, meteorDebug, function(err, out) {
+          projectController.demeteorize(
+            dir, meteorDebug, function (err, out) {
               if (err) {
-                modulus.io.print(err);
+                print(err);
                 return cb(err);
               }
 
               cb(null, out);
             });
-            break;
-        }
-      },
-
-      function (out, cb) {
-        var compressTarget = projectType === Project.types.METEOR
-          ? path.join(out, 'bundle') : out;
-
-        modulus.io.print('Compressing project...');
-        projectController.packageProject(
-          compressTarget,
-          includeModules,
-          function (err, zipFile) {
-            if (err) {
-              // The zip bundle may have been partially created. Remove it.
-              if (zipFile && fs.existsSync(zipFile)) {
-                fs.unlinkSync(zipFile);
-              }
+          break;
+        default:
+          return cb();
+      }
+    },
 
-              return cb('Error compressing project: ' + err);
-            }
+    function (out, cb) {
+      var compressTarget = projectType === Project.types.METEOR
+        ? Path.join(out, 'bundle') : out;
 
-            cb(null, out, zipFile);
-          });
-      },
+      print('Compressing project...');
+      projectController.packageProject(
+        compressTarget,
+        includeModules,
+        function (err, zipFile) {
+          if (err) {
+            // The zip bundle may have been partially created. Remove it.
+            if (zipFile && FS.existsSync(zipFile)) FS.unlinkSync(zipFile);
+            return cb('Error compressing project: ' + err);
+          }
 
-      function(path, zipFile, cb) {
-        projectController._deploy(projectId, zipFile, registry, withTests, function(err, domain) {
-          fs.unlinkSync(zipFile);
+          cb(null, out, zipFile);
+        });
+    },
 
-          //Removes the .demeteorized folder
-          if (projectType === Project.types.METEOR) {
-            fsTools.removeSync(path);
-          }
+    function (path, zipFile, cb) {
+      projectController._deploy(
+        projectId, zipFile, registry, withTests, function (err, domain) {
+          FS.unlinkSync(zipFile);
 
+          //  Removes the .demeteorized folder
+          if (projectType === Project.types.METEOR) FsTools.removeSync(path);
           if (err) {
-            console.log(); //provides a newline
+            print('\n');
```

Comment:
in other locations I have seen `print(' ')` making me think each call injects a new-line, will this result in a double new-line?